Cargo.toml

```diff
@@ -37,3 +37,6 @@ harness = false
 [profile.bench]
 lto = true
 codegen-units = 1
+
+[package.metadata.docs.rs]
+all-features = true
```

Comment:
What is this change supposed to do? Update the generated documentation to show the modules under a feature or? I tried a couple of things and didn't see any difference.

Reply:
This change is for docs.rs, for local builds, `cargo doc --all-features` would generate the same results.

Reply:
`cargo doc --all-features --open` is generating the modules under a feature as well (e.g. mmap), but I assume with your change, plain `cargo doc --open` should do the same, right? It doesn't look like that (at least on my machine), so that's why I'm asking.

Reply:
No, this change is just for [vm-memory docs](https://docs.rs/vm-memory/latest/vm_memory/) in docs.rs, see `[package.metadata.docs.rs]` in the Cargo.toml. Otherwise this docs aren't friendly enough for new lib users.